src/js/common/schemaform/AskVAQuestions.jsx

```diff
@@ -4,7 +4,7 @@ function AskVAQuestions(props) {
   return (
     <div className="row">
       <div className="usa-width-two-thirds medium-8 columns">
-        <div className="help-sidebar-box">
+        <div className="help-footer-box">
```

Comment:
This is now used as a footer-style element (below the main content) rather than on the side.